server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/pipeline_configs/pipeline_config_widget.js.msx

```diff
@@ -140,10 +147,7 @@ define([
 
         var pipelineFlowWidget = function () {
           if(!s.isBlank(ctrl.pipeline().template())) {
```

Comment:
@ketan, do we use lodash at the other places to perform blank string checks? If so, I guess we need to change this for consistency.